src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
+  const cssStyles = {};
+
+  for (let i = 0; i < lines.length; i++) {
+    const line = lines[i].trim();
+
+    if (!line) {
+      continue;
+    }
+
+    const lineSplitted = line.split(':');
+    const [cssProperty, cssValue] = lineSplitted.map((el) => el.trim());
```

Comment:
You're splitting the line by the colon character and then trimming each part. This is a good approach, but you could chain these operations in a more readable format. Also, consider checking for the presence of a colon before splitting to avoid potential errors with lines that don't contain a colon.